vue-frontend/src/components/home-new/PortfolioSection.vue

```diff
@@ -44,21 +44,31 @@
             v-else
             :src="portfolio.images[3]"
             :srcset="`${portfolio.images[0]} 400w, ${portfolio.images[1]} 800w, ${portfolio.images[2]} 1600w, ${portfolio.images[3]} 2400w`"
-            class="tw-w-full tw-object-cover tw-z-[1]"
-            :class="[portfolio.id != 4 ? 'tw-h-full' : 'tw-h-[90%]']"
+            class="tw-object-cover tw-z-[1]"
+            :class="[
+              portfolio.id != 4
+                ? 'tw-h-full'
+                : 'tw-mt-0 sm:tw-mt-4 md:tw-mt-0  sm:tw-w-full tw-h-[90%]',
+            ]"
             :alt="portfolio.title + `-image`"
             loading="lazy"
           />
         </div>
         <div
           v-if="portfolio.description"
-          class="tw-absolute tw-inset-x-0 tw-w-[auto] tw-mx-[-30px] md:tw-mt-[-25px] tw-z-[2] tw-px-0 md:tw-px-2 tw-text-[0.6875rem] tw-leading-[0.875rem] sm:tw-text-[0.75rem] lg:tw-text-[0.875rem] lg:tw-leading-[1rem] xl:tw-text-[1rem] xl:tw-leading-[1.25rem] tw-text-center tw-font-inter-medium"
+          class="tw-absolute tw-inset-x-0 tw-mx-auto md:tw-mt-[-25px] tw-z-[2] tw-text-[0.6875rem] tw-leading-[0.875rem] sm:tw-text-[0.75rem] lg:tw-text-[0.875rem] lg:tw-leading-[1rem] xl:tw-text-[1rem] xl:tw-leading-[1.25rem] tw-text-center tw-font-inter-medium"
           :class="[
             activeIndex == portfolio.id
-              ? 'tw-text-black-core/[0.6] tw-scale-[0.7] sm:tw-scale-[0.8]'
+              ? 'tw--mt-9 tw-text-black-core/[0.87] tw-scale-[0.7] sm:tw-scale-[0.8]'
               : 'tw-text-transparent',
-            portfolio.id == 4
-              ? 'tw-mt-[-10px] lg:tw-mt-[-45px]'
+            portfolio.id == 1
```

Comment:
Ain't we have unique solution for it using flex, instead of checking all indices here?